x/incentives/module.go

```diff
@@ -187,17 +189,20 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 
 // ProposalContents doesn't return any content functions for governance
 // proposals.
+// TODO: Explain this better
 func (AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
 	return nil
 }
 
 // RandomizedParams creates randomized pool-incentives param changes for the
 // simulator.
+// TODO: This should't be nil right?
```

Comment:
TODO: This should't be nil right?

Reply:
Same as above, this is simulator code. Its fine